functions/Export-SqlUser.ps1

```diff
@@ -153,7 +154,7 @@ https://dbatools.io/Export-SqlUser
 
         if ($databases.Count -eq 0)
         {
-            $databases = $sourceserver.Databases | Where-Object {$exclude -notcontains $_.Name -and $_.IsSystemObject -eq $false -and $_.IsAccessible -eq $true}
+            $databases = $sourceserver.Databases | Where-Object {$exclude -notcontains $_.Name -and $_.IsSystemObject -eq $IncludeSystemDatabases -and $_.IsAccessible -eq $true}
```

Comment:
You're right of course. I'm thinking that reverting to the old behavior and then appending system databases (pending accessibility checks and all that) to the list should do the trick.